.gitignore

```diff
@@ -40,3 +40,6 @@ _trial_temp*/
 /docs/build/
 /htmlcov
 /pip-wheel-metadata/
+
+# snapcraft
+*.snap
```

Comment:
this would need to be in the 'filename patterns' section.

Reply:
I've pushed a revised commit that moves this in amongst the file globs - thanks for the guidance, I managed to miss that structure!